spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -20,7 +20,7 @@ content into your application. Rather, pick only the properties that you need.
 	# COMMON SPRING BOOT PROPERTIES
 	#
 	# This sample file is provided as a guideline. Do NOT copy it in its
-	# entirety to your own application.			   ^^^
+	# entirety to your own application.
```

Comment:
That has moved but the purpose is to emphasis the `NOT` part. Can you please rework the PR to restore it (under `NOT`)?

Reply:
@snicoll I suspected so but just removed it as "NOT" is already emphasized by making it upper case and adding "^^^" to the next line of the word to emphasize looks unfamiliar to me. The "unfamiliar" part might be only for me, so if you confirm once more I'll restore and relocate it in the indented position.

Reply:
A double emphasis doesn't hurt as we really don't want users to do that. I agree with the unfamiliar part but that might actually allows us to reach our goal in the end :)